test/comparable_basics_test.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2022, Google Inc. Please see the AUTHORS file for details.
+// All rights reserved. Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import 'package:basics/comparable_basics.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Comparable', () {
```

Comment:
Nit: The other test files are using a structure of the group name being the method name (since the test files are already per extension, so the group name being the extension is redundant), and the test names being individual behaviors. I think it's fine to lump all the operators together as one group, but can you update the rest of the tests here to match the other test files a bit more?

Reply:
I'll just remove the `group`, which was an artifact from an earlier incarnation where these weren't in a separate Dart library.